config/database.go

```diff
@@ -75,12 +78,18 @@ func NewDatabaseStore(dsn string) (ds *DatabaseStore, err error) {
 //
 // Uses MEDIUMTEXT on MySQL, and TEXT on sane databases.
 func initializeConfigurationsTable(db *sqlx.DB) error {
+	var count int
+	err := db.Get(&count, `SELECT COUNT(*) FROM `+migrationsTableName+` WHERE Name = 'create_configurations'`)
+	if err == nil && count != 0 {
```

Comment:
Can we return the error if it's not nil, and only proceed if err == nil && count != 0?

```go
if err != nil {
  return errors.Wrap(err, "...")
} else if count != 0 {
  return nil
}
```

Reply:
In case there is a schema before morph, this will return an error since this wouldn't find the table.